tests/model_registry/python_client/test_model_registry_creation_oauth.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+from typing import Self
+from simple_logger.logger import get_logger
+from pytest_testconfig import config as py_config
+
+from ocp_resources.namespace import Namespace
+from utilities.constants import DscComponents, Protocols
+from tests.model_registry.constants import MODEL_NAME, MODEL_DICT
+from model_registry import ModelRegistry as ModelRegistryClient
+from tests.model_registry.utils import get_endpoint_from_mr_service, get_mr_service_by_label
+
+LOGGER = get_logger(name=__name__)
+
+
+@pytest.mark.parametrize(
+    "updated_dsc_component_state_scope_class",
+    [
+        pytest.param(
+            {
+                "component_patch": {
+                    DscComponents.MODELREGISTRY: {
+                        "managementState": DscComponents.ManagementState.MANAGED,
+                        "registriesNamespace": py_config["model_registry_namespace"],
+                    },
+                },
+            },
+        ),
+    ],
+    indirect=True,
+)
+@pytest.mark.usefixtures("updated_dsc_component_state_scope_class")
+class TestModelRegistryCreationOAuth:
+    """
+    Tests the creation of a model registry with OAuth proxy configuration.
+    """
+
+    @pytest.mark.smoke
```

Comment:
TestDevops still uses `latest` tag for all versions. So you may want to hold off adding it to smoke for the time being. May be we should have a jira to track and enable it once TestDevops starts using right images for right versions.

Reply:
Mh, we do want this to be run though. Perhaps we can add a dynamic check here on the installed version and execute only if >=2.21?

Reply:
Let's do that in a separate PR. The reason being, your PR is ready to be merged. I don't want to have to wait for it, just because of this marker.